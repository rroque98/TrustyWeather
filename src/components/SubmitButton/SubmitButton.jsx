import React from "react";

const SubmitButton = ({ value, handleClick }) => (
  <div>
    <input type="submit" value={value} onClick={handleClick} />
  </div>
);

export default SubmitButton;
