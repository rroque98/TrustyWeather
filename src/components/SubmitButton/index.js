export { default } from "./SubmitButton.jsx";
