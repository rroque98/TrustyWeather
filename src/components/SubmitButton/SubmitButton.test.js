import React from "react";
import { render, cleanup } from "@testing-library/react";
import SubmitButton from "./index.js";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<SubmitButton />);
  expect(asFragment()).toMatchSnapshot();
});
