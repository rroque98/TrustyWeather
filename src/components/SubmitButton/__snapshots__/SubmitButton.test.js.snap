// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <input
      type="submit"
    />
  </div>
</DocumentFragment>
`;
